rust/hyperlane-base/src/types/checkpoint_syncer.rs

```diff
@@ -116,4 +116,23 @@ impl CheckpointSyncer for CheckpointSyncers {
             CheckpointSyncers::S3(syncer) => syncer.write_checkpoint(signed_checkpoint).await,
         }
     }
+
+    #[instrument(err, skip(self))]
+    /// Write the signed announcement to this syncer
+    async fn write_announcement(&self, signed_announcement: SignedAnnouncement) -> Result<()> {
```

Comment:
Can you just use a borrowed `SignedAnnouncement` here? (we should have probably done the same with the `SignedCheckpoint`

Reply:
Yeah we can and IIUC should pass by reference, done